velox/exec/tests/RowContainerTest.cpp

```diff
@@ -937,24 +1302,80 @@ TEST_F(RowContainerTest, alignment) {
   }
 }
 
-// Verify comparison of fringe float valuesg
+// Verify comparison of fringe float values
 TEST_F(RowContainerTest, compareFloat) {
   // Verify ascending order
-  testCompareFloats<float>(REAL(), true, true);
-  testCompareFloats<float>(REAL(), true, false);
+  testCompareSimpleFloats<float>(REAL(), true, true);
```

Comment:
There is a lot of copy-paste here. Consider introducing a helper function:

```
  template <typename T>
  void testCompareSimpleFloats(const TypePtr& type) {
    // Verify ascending order.
    testCompareSimpleFloats<float>(REAL(), true, true);
    testCompareSimpleFloats<float>(REAL(), true, false);
    // Verify descending order.
    testCompareSimpleFloats<float>(REAL(), false, true);
    testCompareSimpleFloats<float>(REAL(), false, false);
  }

TEST_F(RowContainerTest, compareFloat) {
  testCompareSimpleFloats<float>(REAL());
}

TEST_F(RowContainerTest, compareDouble) {
  testCompareSimpleFloats<double>(DOUBLE());
}
```

Also, add SCOPED_TRACE to allow for easier troubleshooting of test failures.

```
  template <typename T>
  void testCompareSimpleFloats(TypePtr type, bool ascending, bool nullsFirst) {
    SCOPED_TRACE(fmt::format("{}, ascending = {}, nullsFirst = {}", type->toString(), ascending, nullsFirst));
```

Same for arrays, maps and structs.